modules/builtin/src/content-types/_utils.js

```diff
@@ -37,8 +37,19 @@ function extractFileName(file) {
 }
 
 function extractPayload(type, data) {
-  // for channel renderers
-  return { type, ..._.pickBy(_.omit(data, 'event', 'temp', 'user', 'session', 'bot', 'BOT_URL'), v => v !== undefined) }
+  const payload = {
+    type,
+    ...data
+  }
+
+  delete payload.event
+  delete payload.temp
+  delete payload.user
+  delete payload.session
+  delete payload.bot
+  delete payload.BOT_URL
```

Comment:
Just wondering, why the change? omit did make the code clearer

Reply:
I required lodash and I was getting problems with the `require` so I removed it.